source/includes/gho/_introduction.md

```diff
@@ -40,6 +40,63 @@ Using the Greenhouse Onboarding API provides access to _all_ of your company's i
 the scope of an API key.  Only share your API key with people that you trust.  API keys can be revoked at any time
 on the API Management screen.
 
-## Resource Limitations
+## Rate Limiting
 
-Be kind.
+The Greenhouse Onboarding API imposes limits on the the number of requests a single client can send us, as well as the
+complexity of these requests.  This is done to prevent abusive or misbehaving clients from affecting the performance
```

Comment:
Don't accuse customers of misbehavior. I'd put something like "This ensures the API remains performant for all our customers" or something like that.